sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/UserTest.java

```diff
@@ -124,8 +134,57 @@
     };
   }
 
+  @DataProvider(name = "getUserByLoginIdFailure")
+  public Object[][] getUserByLoginIdFailure() {
```

Comment:
Place request.json and response.json (and any other assets) in a separate folder for each test.

user.get.byloginid

testGetUserByLoginIdFailureWithInvalidLoginId
testGetUserByLoginIdFailureWithEmptyLoginId
testGetUserByLoginIdFailureWithoutLoginId